deploy/crds/alerts_v1alpha1_storagealert_cr.yaml

```diff
@@ -1,11 +1,48 @@
 apiVersion: alerts.monstorak.org/v1alpha1
 kind: StorageAlert
+metadata:
+  name: ceph-storagealert
+  namespace: storage-monitoring
+spec:
+  storage:
+  - provider: "ceph"
+    version:
+    - "v14.2.1-20190430"
+    namespace: "openshift-storage"
+  prometheus:
+    namespace: "openshift-monitoring"
+    label: "prometheus:k8s"
+---
+apiVersion: alerts.monstorak.org/v1alpha1
+kind: StorageAlert
+metadata:
+  name: nooba-storagealert
+  namespace: storage-monitoring
+spec:
+  storage:
+  - provider: "nooba"
+    version:
+    - "v1.0"
+    namespace: "openshift-storage"
+  prometheus:
+    namespace: "openshift-monitoring"
+    label: "prometheus:k8s"
```

Comment:
label: "prometheus:k8s" // ? trying to understand the label here

Reply:
Will there be only one prometheus instance for multiple storages?

Reply:
>label: "prometheus:k8s" // ? trying to understand the label here

That is the label of the prometheus instance to be selected.

> Will there be only one prometheus instance for multiple storages?

We will have one prometheus for multiple storages. But, you can create another object that makes use of another instance of prometheus (But, this is not suggested)